helper.mk

```diff
@@ -86,6 +86,15 @@ setup/debian/stretch: /etc/apt/sources.list /etc/os-release
 	${sudo} apt-get update
 	${sudo} apt-get install -y ${packages}
 
+setup/debian/buster: /etc/apt/sources.list /etc/os-release
+	grep 'VERSION="10 (buster)"' /etc/os-release
+#	${sudo} sed -e 's|\(http://\)\(.*\)\(.debian.org\)|\1archive\3|g' -i $<
+#	${sudo} sed -e 's|stretch-updates|stretch-proposed-updates|g' -i $<
+#	echo "deb http://archive.debian.org/debian stretch-backports main contrib non-free" \
+| ${sudo} tee "$<.d/backports.list"
+	${sudo} apt-get update
+	${sudo} apt-get install -y ${packages}
+
 setup/debian: setup/debian/stretch
```

Comment:
that line should be changed too setup/debian/buster and probably other files, like:

https://github.com/SiliconLabs/zipgateway/blob/main/Dockerfile#L4